tests/test_custom_components.py

```diff
@@ -1,70 +1,24 @@
 import dash_ag_grid as dag
 from dash import Dash, html, Input, Output
-import pandas as pd
-import yfinance as yf
 from . import utils
 import json
-import os
-import time
+
 
 def test_cc001_custom_components(dash_duo):
 
-    app = Dash(__name__,
```

Comment:
I would prefer leaving the meta tags in here, so that we make sure the components load even when the application is locked down.